rand_jitter/src/lib.rs

```diff
@@ -769,9 +765,27 @@ where
         // themselves via `fill_bytes`.
         impls::fill_bytes_via_next(self, dest)
     }
+}
+
+impl<F> rand_core::TryRngCore for JitterRng<F>
+where
+    F: Fn() -> u64 + Send + Sync,
+{
```

Comment:
We can't use `impl_try_rng_from_rng_core` here. Ideally it would support types with generics, but probably not worth bothering.